ocp/cosa-image.yaml

```diff
@@ -0,0 +1,220 @@
+# In this file, you will find the following:
+#   - ImageStream configs for importing Fedora images
+#   - ImageStream configs for importing coreos-assembler images
+#   - BuildConfig defining how coreos-assembler:master is built
+#   - BuildConfig defining how coreos-assembler:buildconfig is built
+#
+# To use this file, you would do something like the following:
+#   $ oc apply -f cosa-image.yaml
+#
+# It is possible to provide parameters used by the BuildConfig, e.g.:
+#   $ oc process -f cosa-image.yaml --param=REPO_REF=foo \
+#                                   --param=NAMESPACE=bar | \
+#     oc apply -f -
+#
+apiVersion: v1
+kind: Template
+labels:
+  app: cosa
+  template: cosa-template
+metadata:
+  annotations:
+    description: CoreOS Assembler Images
+    openshift.io/display-name: CoreOS Assembler
+    tags: cosa,coreos-assembler
+  name: cosa-images
+
+parameters:
+  - description: Git source URI
+    name: REPO_URL
+    value: "https://github.com/darkmuggle/coreos-assembler"
+  - description: Git branch/tag reference
+    name: REPO_REF
+    value: "psi"
```

Comment:
Looks obsolete

Reply:
Yeah, this is my "testing" push space. :)